src/win32/compat.c

```diff
@@ -145,6 +146,70 @@ inet_aton(const char *AddrStr, struct in_addr *Addr)
 }
 #endif /* !defined(MINGW_VERSION) || MINGW_VERSION < 40600 */
 
+/* clock_gettime() is not available on Win32 (unless we link libwinpthread)
+ * this clock_gettime() and dependency getFILETIMEoffset() is based on
+ * http://stackoverflow.com/questions/5404277/porting-clock-gettime-to-windows
+ * thanks to http://stackoverflow.com/users/658776/carl-staelin
+ */
+void
+getFILETIMEoffset(LARGE_INTEGER *t)
```

Comment:
Please explain why this is needed.
My windows builds run just fine on MINGW (on Linux)